tests/improver-generate-topographybands-ancillary/03-bounds_json.bats

```diff
@@ -34,19 +34,20 @@
 @test "generate-topographybands-ancillary input_orog.nc input_land.nc output.nc" {
   TEST_DIR=$(mktemp -d)
   improver_check_skip_acceptance
-  test_path=$IMPROVER_ACC_TEST_DIR/generate-topographybands/basic/
+  #test_path=$IMPROVER_ACC_TEST_DIR/generate-topographybands/basic/
+  test_path=~/IMP-376-DATA/
```

Comment:
Opps, you've left this in from your testing. Please can you change back (even though the data isn't in there yet).

Reply:
Yes.